### PR TITLE
Upgrade jest and switch to inline snapshots

### PR DESCRIPTION
**Pull request by k4b7**
## Summary:
It was very hard to see what was going on with the current tests b/c all of the snapshots were in a separate file.  Since the snapshots are all pretty short, it makes sense to have them inline.  This require an upgrade of jest to v23 (I didn't upgrade past that b/c I ran into issue that I don't want to bother digging into at the moment and v23 supports inline snapshots) and adding prettier (which is required for inline snapshots).

These are essentially the same changes I made in #6, but I wanted to pull them out into a separate PR to make things easier to review.

Issue: None

## Test plan:
- yarn test

### CODE REVIEW

**Review by jeresig — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
     "babel-preset-react": "^6.24.1",
     "express": "^4.16.1",
     "flow-bin": "^0.41.0",
-    "jest": "^21.2.1",
+    "jest": "23",
```
Comment (somewhatabstract) [inline package.json:33]: **question:** Only 23? Why not the latest?

**Review by somewhatabstract — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (somewhatabstract) [reply package.json:33]: Oh, I see you address that in the PR description. 👍🏻

Comment (k4b7) [reply package.json:33]: I ran into issues upgrading to the latest.  I'll try again and post the errors.

Comment (k4b7) [reply package.json:33]: The error I was running into was the following:
```
 FAIL  __tests__/test.js
  ● Test suite failed to run

    Plugin/Preset files are not allowed to export objects, only functions. In /Users/kevinbarabash/khan/react-loadable/node_modules/babel-preset-es2015/lib/index.js

      at createDescriptor (node_modules/@babel/core/src/config/config-descriptors.ts:350:11)
          at createDescriptor.next (<anonymous>)
      at evaluateSync (node_modules/gensync/index.js:251:28)
      at node_modules/gensync/index.js:31:34
          at Array.map (<anonymous>)
      at Function.sync (node_modules/gensync/index.js:31:22)
      at Function.all (node_modules/gensync/index.js:210:24)
      at createDescriptors (node_modules/@babel/core/src/config/config-descriptors.ts:261:38)
          at createDescriptors.next (<anonymous>)
```

**Pull request merged**